elytron/WFCORE-3666-alternative-rolemapping.adoc

```diff
@@ -0,0 +1,82 @@
+= WFCORE-3666 Provide Elytron alternative to RoleMappingLoginModule
+:author:            Martin Mazanek
+:email:             mmazanek@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently, Elytron role mapping consist mostly of logical and basic add suffix/prefix RoleMappers, which makes it hard to map roles based on an actual map. This task is to introduce more ways of mapping roles, such as using a map or regex, without the customers needing to implement their own custom components.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-3666[WFCORE-3666]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ELY-1591[ELY-1591]
+* https://issues.jboss.org/browse/EAP7-1008[EAP7-1008]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
```

Comment:
please fill

Reply:
Will do, although in my case this works as intended